### ISSUE TITLE
[bug]: ModuleNotFoundError: No module named 'torch'

### ISSUE THREAD

**Issue by vikas5914** [invoke-ai/InvokeAI#1409]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

10 GB

### What happened?

Getting following error while running: `pew new --python 3.10 -r requirements-lin-win-colab-CUDA.txt invokeai`

### Screenshots

![image](https://user-images.githubusercontent.com/7912297/200419303-2d450ba0-213f-40a8-88b0-d77c03a74b1c.png)

**Comment by addianto** [comment]
Hello @vikas5914,

> ### What happened?
> 
> Getting following error while running: `pew new --python 3.10 -r requirements-lin-win-colab-CUDA.txt invokeai`

I encountered the same error when installing InvokeAI using `pip` in a virtual environment (`venv`).
I was able to resolve it by installing `torch` and `torchvision` packages prior to installing all required packages.

Since I didn't follow the easy-peasy guide when setting up InvokeAI for the first time, the following procedures might need to be adjusted to `pew`:

Try to set up a new virtual environment and activate it prior to installing packages:

```shell
python -m venv .venv
# If you are using Windows:
.venv\Scripts\activate
```

Then, use `pip` to install `torch` and `torchvision` manually with the exact version listed in the requirements file:

```python
pip install --extra-index-url https://download.pytorch.org/whl/cu116 --trusted-host https://download.pytorch.org torch==1.12.1 torchvision==0.13.1
```

Then, install all required packages listed in `requirements-lin-win-colab-CUDA.txt`:

```python
pip install -r requirements-lin-win-colab-CUDA.txt
```

I hope that helps! 😃

**Comment by vikas5914** [comment]
@addianto I tried to add that into `requirements.txt.` and it installs but still getting the error since `basicsr-1.4.1` has dependency hardcoded to the old version.

**Comment by addianto** [comment]
> @addianto I tried to add that into `requirements.txt.` and it installs but still getting the error since `basicsr-1.4.1` has dependency hardcoded to the old version.

Sorry if I was not clear enough in my previous message. You don't need to modify `requirements.txt` on your side. 

Before calling `pip install -r requirements.txt`, you need to execute the following command:

```shell
pip install --extra-index-url https://download.pytorch.org/whl/cu116 --trusted-host https://download.pytorch.org torch==1.12.1 torchvision==0.13.1
```

Once you done that, run `pip install --prefer-binary -r <the requirements.txt file>`

**Comment by robotdad** [comment]
Another thing you might be running into is Python 3.11 is not compatible with the pinned version of Torch. I just updated today and realized looking at the downloads here that Torch 1.12.1 is only available up to Python 3.10.

**Comment by mickr777** [comment]
Side note: this also occurs on ubuntu 22.04 and using requirements-lin-cuda.txt,  however I have to install torch 1.13 and torchvision 1.14 or I get incompatible cuda card, (rtx3060 12gb)
